elastic_transport/_node/_http_aiohttp.py

```diff
@@ -182,7 +182,7 @@ async def perform_request(  # type: ignore[override]
                 data=body_to_send,
                 headers=request_headers,
                 timeout=aiohttp_timeout,
-                **kwargs,
+                **kwargs,  # type: ignore[arg-type]
```

Comment:
This is not related to Python 3.13, but caused instead by https://github.com/aio-libs/aiohttp/pull/8463. I.. just have no idea how to make mypy happy here, short of maybe adding a class?